data_pipeline/algolia-update.js

```diff
@@ -0,0 +1,43 @@
+/* eslint-disable @typescript-eslint/no-var-requires */
+const fs = require('fs');
+const dotenv = require('dotenv');
+const algoliaSearch = require('algoliasearch');
+
+//load local environment variables
+const env = {
+  local: dotenv.parse(fs.readFileSync('.env.local'))
+};
+
+const charities = JSON.parse(
+  fs.readFileSync('data_pipeline/charity.json').toString('utf8')
+);
+
+const updatedAlgoliaIndex = Object.entries(charities).map(
```

Comment:
how does this make you feel leave giffy below

Reply:
![](https://media.giphy.com/media/Jlt69Ka6SwtH2/giphy.gif)